CHANGES.rst

```diff
@@ -210,9 +210,8 @@ API Changes
 
 - ``astropy.utils``
 
-  - Removed the deprecated ``astropy.utils.compat.subprocess`` module. [#5975]
-
-  - Removed the deprecated ``astropy.utils.compat.odict`` module. [#6157]
+  - Removed the deprecated compatibility modules for Python 2.6 (``argparse``,
+    ``fractions``, ``gzip``, ``odict``, ``subprocess``) [#5975,#6157,#6164]
 
```

Comment:
I merged these lines to make it more digest, hope it's fine for the release scripts @bsipocz ?